lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -38,9 +38,12 @@
   <div class="data-display-table" id="data-student-profiles-table"></div>
 
   %if settings.FEATURES.get('ALLOW_COURSE_STAFF_GRADE_DOWNLOADS') or section_data['access']['admin']:
-    <p>${_("Click to generate a CSV grade report for all currently enrolled students.")}</p>
+    <p>${_("Click to generate a CSV grade report for all currently enrolled students, or a CSV submissions report for all problems.")}</p>
```

Comment:
@sarina Agreed. My inclination is to make that text more generic. Let me consult with @explorerleslie.